ui/src/app/course/pages/list/page-list.component.spec.ts

```diff
@@ -37,9 +37,10 @@ describe('CoursePageListComponent', () => {
 		fixture.detectChanges();
 	});
 
-	it('listCourses', () => {
-		expect(component.listCourses).toEqual(arr)
-	});
+	// TODO: need implement this test
```

Comment:
Try to use mock for input and check the property listCourses with the same value.